SRIJAN PRATAP SRIVASTAVA/IF-ELSE/discount.cpp

```diff
@@ -0,0 +1,14 @@
+#include<iostream>
+using namespace std;
+int main()
+{
+    int quantity;
+    cout<<"enter the quantity:";
+    cin>>quantity;
+    if (quantity>1000)
```

Comment:
In question we have to check is total cost is greater than 1000 or not, not quantity. So you need to check if condition like `if(quantity*100 > 1000)`, assuming cost as 100.